docker-compose.yml

```diff
@@ -8,6 +8,7 @@ services:
       args:
         GIT_COMMIT: ${GIT_COMMIT-local-dev}
         PIPENV_DEV: "True"
+        USER_ID: ${USER_ID-1000}
```

Comment:
I added this to allow making the UID of the `koku` user match the user that owns the directory on the contraire host. This was an issue when using 9p and Rancher Desktop but wasn't necessary with Docker Desktop using grpcfuse.